test/server/lib/validation/index.test.js

```diff
@@ -19,71 +18,61 @@ describe.skip('server/lib/validation/applications', () => {
     done();
   });
 
-  it('should validate a correctly formatted contact name', () => {
-    expect(validator.isValidContactName('valid name')).to.eventually.equal(true);
-    expect(validator.isValidContactName('valid hyphenated-name')).to.eventually.equal(true);
-    expect(validator.isValidContactName('valid quoted\'name')).to.eventually.equal(true);
+  it('should validate a correctly formatted app  name', () => {
+    expect(validator.isValidAppName('app name')).to.equal(true);
+    expect(validator.isValidAppName('valid hyphenated-name')).to.equal(true);
+    expect(validator.isValidAppName('valid quoted\'name')).to.equal(true);
     expect(stubLogger).to.have.been.calledThrice;
   });
-
-  it('should validate and return an error if contact name is not correctly formatted', () => {
-    const errors = {};
-    errors.fullName = errorManifest.fullName.incorrect;
-    expect(validator.isValidContactName('-日女子أَبْجَدِيّ')).to.be.rejectedWith(errors);
+  it('should validate and return an error for blank app name', () => {
+    expect(validator.isValidAppName('')).to.equal(false);
     expect(stubLogger).to.have.been.calledOnce;
   });
 
-  it('should validate and return an error for blank contact name', () => {
-    const errors = {};
-    errors.fullName = errorManifest.fullName.blank;
-    expect(validator.isValidContactName('')).to.be.rejectedWith(errors);
-    expect(validator.isValidContactName(undefined)).to.be.rejectedWith(errors);
-    expect(validator.isValidContactName(null)).to.be.rejectedWith(errors);
+  it('should validate a correctly formatted app description', () => {
+    expect(validator.isValidDescription('Lorem ipsum dolor sit amet consectetur adipiscing elit.')).to.equal(true);
+    expect(validator.isValidDescription('Lorem-ipsum HYPHEN allowed.')).to.equal(true);
```

Comment:
Perhaps a more readable way of putting these would be
expect(validator.isValidDescription('Lorem-ipsum HYPHEN allowed.')).to.be.true